pkg/cli/admin/release/bug.go

```diff
@@ -278,6 +294,7 @@ func extractRefs(msg string) (RefList, string) {
 	if bt == Jira {
 		refs = strings.TrimSpace(parsedMsg[0])
 		refs = strings.TrimSuffix(refs, ":")
+		refs = strings.TrimSuffix(refs, "-")
```

Comment:
Since you control these naming patterns, I assume this change is safe

Reply:
AFAIK, the Jira automation, that we have in place, only expects the `:`.  The regex used in `oc`, for Bugzilla and Jira, allows for both `:` and `-`.  
In validating these changes, I found a bunch of occurrences where the changelog had been incorrectly parsing Jira tickets that used the `-`, as seen here:
![JiraParsingIssue](https://user-images.githubusercontent.com/43015045/212089952-7bf36732-0d35-4dca-921c-9f696dc82183.png)
Instead of ignoring these tickets all together (by changing the regex), I chose to simply trim off the `-` if when it appears.